src/components/Form/Form.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { IoMdCheckmark } from "react-icons/io";
+
+const Form = () => {
+  return (
+    <form className="search-form">
+      <div className="search-form--specification">
+        <label for="ingredient" className="search-form__label">
+          Ingredient:
+        </label>
+        <input
+          id="ingredient"
+          className="search-form__input"
+          placeholder="eggs, spinach"
+          type="text"
+        />
+        <ul className="search-form--ingredient-list"></ul>
+      </div>
+      <div className="search-form__checkbox--container">
+        <h3 className="search-form__checkbox--instructions">
+          Click to select dietary restrictions:
+        </h3>
+        <div className="search-form__checkboxes">
+          <div className="search-form__checkbox">
```

Comment:
I notice this div class name is called per each dietary input. Does it have to be written for each input or can we reduce this to one with all input in the body?